### PR TITLE
hide ID column in patients list specifically for eproms

### PR DESCRIPTION
**Pull request by achen2401**
address this story:  
https://www.pivotaltracker.com/story/show/149439243

- hide ID column in patients list specifically for EPROMs
- add row link a tag as ID column is hidden (need a tag with href to navigate to patients profile page)
- remove unused and unnecessary code (e.g. loader fadeout code, etc.)

### CODE REVIEW

**Comment by achen2401** [comment]
**Wait to merge** this after the RC has created.

**Comment by achen2401** [comment]
can be merged upon review now that RC branch has been created.

File: portal/static/js/main.js [hunk 1]
```diff
@@ -3729,6 +3729,19 @@ var tnthTables = {
         //if (aa < bb) return -1;
         //return 0;
         return  bb - aa;
+    },
+    /***
+     * Quick way to sort when text is wrapper in an <a href> or other tag
```
Comment (pbugni) [inline portal/static/js/main.js]: I imagine you mean `wrapped` ?

File: portal/static/js/main.js [hunk 2]
```diff
@@ -3729,6 +3729,19 @@ var tnthTables = {
...
+     * NOTE for text that is NOT number
```
Comment (pbugni) [inline portal/static/js/main.js]: Why `NOT number`?  (I assume as it'll sort lexicographically not numerically)?

Comment (achen2401) [inline portal/static/js/main.js]: Yeah, Paul, it is a function specifically for sorting lexicographically.

**Comment by achen2401** [comment]
requirement changed for this.  closing this now.

**Pull request closed without merging**